looper/__init__.py

```diff
@@ -39,7 +40,14 @@
 # the locations of some of the peppy declarations. Effectively, concentrate
 # the connection between peppy and looper here, to the extent possible.
 
-__all__ = ["Project", "PipelineInterface", "SubmissionConductor"]
+__classes__ = ["ComputingConfiguration"]
+__functions__ = ["select_divvy_config"]
+__all__ = (
+    ["Project", "PipelineInterface", "SubmissionConductor"]
+    + __classes__
+    + __functions__
+    + [write_submit_script.__name__]
```

Comment:
is there a reason do this instead of "write _submit_script" -- why is this different from the "select_divvy_config" function?

Reply:
Yeah, I'm not 100%, I was preserving what was found in the init for divvy

https://github.com/pepkit/divvy/blob/ef51a9e5225b7ce1b8dd36713442687816298422/divvy/__init__.py#L12-L18

Reply:
I removed "write submit" this is good now